src/custom/components/swap/TradePrice/TradePriceMod.tsx

```diff
@@ -73,7 +72,7 @@ export default function TradePrice({ price, showInverted, fiatValue, setShowInve
       }}
       // title={text}
     >
-      <Text fontWeight={500} fontSize={14} color={theme.text1}>
+      <Text fontWeight={500} fontSize={13} color={theme.text1}>
```

Comment:
🤷‍♂️

Reply:
Because before we had a style overrides, I got rid of this